tests/tests.py

```diff
@@ -228,6 +228,10 @@ def test_ancient():
     run(dpath("test_ancient"), targets=["D", "C", "old_file"])
 
 
+def test_ancient_cli():
```

Comment:
perhaps run all the tests also without the `ancient` flag to "prove" that the output is then generated?

Reply:
Done